sagemaker-experiments/sagemaker_job_tracking/tensorflow_script_mode_training_job.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>Use MNIST dataset from sagemaker-sample-files instead of from Keras - that way if Keras moves or delete the dataset, this notebook will not be affected</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>